unit/testing-utils/load_java_class.cpp

```diff
@@ -28,15 +28,17 @@
 /// \return The symbol table that is generated by parsing this file.
 symbol_tablet load_java_class(
   const std::string &java_class_name,
-  const std::string &class_path)
+  const std::string &class_path,
+  const std::string &main)
```

Comment:
Update the documentation with an explanation of the default behaviour (`""` for `main`)